src/sempy_labs/migration/_migrate_calctables_to_lakehouse.py

```
@@ -101,168 +100,134 @@ def migrate_calc_tables_to_lakehouse(
-                                                    and c.Name == new_column_name
-                                                )
-
-                                            if dataType == "Int64":
-                                                df[new_column_name] = df[
-                                                    new_column_name
-                                                ].astype(int)
-                                            elif dataType in ["Decimal", "Double"]:
-                                                df[new_column_name] = df[
-                                                    new_column_name
-                                                ].astype(float)
-                                            elif dataType == "Boolean":
-                                                df[new_column_name] = df[
-                                                    new_column_name
-                                                ].astype(bool)
-                                            elif dataType == "DateTime":
-                                                df[new_column_name] = pd.to_datetime(
-                                                    df[new_column_name]
-                                                )
-
-                                        delta_table_name = t.Name.replace(
-                                            " ", "_"
-                                        ).lower()
-
-                                        spark_df = spark.createDataFrame(df)
-                                        filePath = create_abfss_path(
-                                            lakehouse_id=lakehouse_id,
-                                            lakehouse_workspace_id=lakehouse_workspace_id,
-                                            delta_table_name=delta_table_name,
+                                    except Exception:
+                                        dataType = next(
```

what kind of exception do you expect here?

---

It's masking the Exception. I will update this in another PR with a better method.